src/recoil_values/Recoil_atomFamily.js

```diff
@@ -177,18 +177,18 @@ function atomFamily<T, P: Parameter>(
   let legacyAtom;
   // prettier-ignore
   // @fb-only: if (
-    // @fb-only: options.scopeRules_APPEND_ONLY_READ_THE_DOCS
+  // @fb-only: options.scopeRules_APPEND_ONLY_READ_THE_DOCS
```

Comment:
unfortunately these are generated by the build system that is patching the code while syncing it to github and should probably not be touched. do you mind reverting the formatting?